README.md

```diff
@@ -31,23 +34,59 @@ To run in watch mode, add `-w`/`--watch` to the command:
 steiger ./src --watch
 ```
 
-# Configuration
+## Configuration
 
 Steiger is zero-config! If you don't want to disable certain rules, you can safely skip this section.
 
 Steiger is configurable via `cosmiconfig`. That means that you can create a `steiger.config.ts` or `steiger.config.js` file in the root of your project to configure the rules. Import `{ defineConfig } from "steiger"` to get autocompletion.
 
-```ts
-import { defineConfig } from 'steiger'
+The config file shape is highly inspired by ESLint's config file, so if you have configured ESLint before, you'll find it easy to configure Steiger.
+
+### Example
+
+Here are some rules on how configuration is processed:
+
+- Config objects are processed from top to bottom, so if there are multiple config object that match the same file for the same rule, the last one will be applied.
+- You can set options for a rule once. When set, options are applied for the entire file system that is covered by Steiger.
+
+Note that this line `...fsd.configs.recommended,` just takes the plugin and the recommended rules configuration (all enabled with "error" severity by default) and puts it into the config array.
 
-export default defineConfig({
-  rules: {
-    'public-api': 'off',
+```javascript
+// ./steiger.config.ts
+import fsd from '@feature-sliced/steiger-plugin'
+import defineConfig from 'steiger'
+
+export default defineConfig([
+  ...fsd.configs.recommended,
+  {
+    // ignore all mock files for all rules
+    ignores: ['**/__mocks__/**'],
+  },
+  {
+    files: ['**/shared/**'],
```

Comment:
suggestion (non-blocking): ideally I think we should encourage people to provide precise paths to layers, otherwise if there's a folder called "shared" somewhere deep within, it will also be targeted
```suggestion
    files: ['./src/shared/**'],
```